diagnostics/ENSO_MSE/COMPOSITE/COMPOSITE.py

```diff
@@ -301,30 +299,75 @@
 ##################################################3
 #############   El Nino/La Nina indices selection
 
-ii1, ii2, jj1, jj2, ttmax1, years1, ttmax2, years2 = get_nino_index(imax, jmax, lon, lat,  itmax,  iy1, iy2, imindx1, imindx2, llon1, llon2, llat1, llat2, ii1, ii2, jj1, jj2, sigma, ttmax1, ttmax2, years1,  years2,  prefix1, undef2)
-
+ii1, ii2, jj1, jj2, ttmax1, years1, ttmax2, years2 = get_nino_index(imax, jmax, lon, lat,  itmax,  iy1, iy2, imindx1, imindx2, llon1, llon2, llat1, llat2, ii1, ii2, jj1, jj2, sigma, ttmax1, ttmax2, years1,  years2,  prefix1, undef)
+
+## added 2020-09-08  check for the selected years - the time series needs to be long enough
+##  to capture El Nino/La Nina events
+if( ttmax1 <= 0 ):
+    print ("WARNING: The number of  El Nino events is : ", ttmax1 )
+    print ("At least 1 event is needed for calculation ")
+    print ("and at least 2 events for sucessfull completion")
+    print ("Please, extend the time span of your Model data")
+    #print ("The scripts is exiting  after pressing any key")
+    #input("Press any key to continue")
+    exit()
+
+if( ttmax1 == 1 ):
+    print ("WARNING: The number of  El Nino events is : ", ttmax1)
+    print ("At least 2 events are needed for successful  calculations ")
+    print ("The code will run just with 1 event, but with limited results")
+    print ("You may  extend the time span of your Model data")
+    #print ("Please, if you wish to continue  then : ")
+    #input("Press any key to continue")
+
+if( ttmax1 >= 2 ):
+    print ("The number of  El Nino events is : ", ttmax1)
+    print ("The code will proceed  with this number of El Nino events")
+    time.sleep(4.)
+ 
+if( ttmax2 <= 0 ):
+    print ("WARNING: The number of La Nina events is : ",  ttmax2)
+    print ("At least 1 event is needed for calculation ")
+    print ("and at least 2 events for sucessfull completion")
+    print ("Please, extend the time span of your Model data")
+    #print ("The scripts is exiting  after pressing any key")
+    #input("Press any key to continue")
+    exit()
+
+if( ttmax2 == 1 ):
+    print ("WARNING: The number of La Nina events is : ", ttmax2)
+    print ("At least 2 events are needed for successful  calculations ")
+    print ("The code will run just with 1 event, but with limited results")
+    print ("You may  extend the time span of your Model data")
+    #print ("Please, if you wish to continue  then : ")
+    #input("Press any key to continue")

```

Comment:
Are the commented lines necessary?

Reply:
commented out lines are removed.  regarding consolidation of similar code I will look into that